Documentation/ApiOverview/ContentElements/Index.rst

```diff
@@ -110,21 +110,86 @@ your projects.
 possible without writing PHP code and can therefore also be done by
 TYPO3 integrators.
 
+..  _plugins:
 
+Plugins in TYPO3
+----------------
+
+A **plugin** in TYPO3 is a more complex module, typically providing dynamic
+or interactive functionality. Plugins are usually provided by extensions
+that introduce new features to the website.
+
+The data to be displayed is usually supplied by a special PHP class
+called a "controller". Depending on the technology used in the controller
+the plugin can be an Extbase plugin or a plain plugin.
```

Comment:
There are no controllers in abstract plugin implementation.

```suggestion
The data to be displayed can be realized by an Extbase plugin 
following the MVC pattern or with help of a very basic plugin 
structure called "Abstract Plugin" where you have full access to
TYPO3 API.
```

Reply:
The AbstractPlugin rests in peace: https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/12.4/Deprecation-100639-DeprecateAbstractPlugin.html#deprecation-100639-deprecate-abstractplugin and you should implement a Controller pattern in non-extbase plugins, though noone can force you to...